packages/parser-javascript/package.json

```diff
@@ -7,12 +7,13 @@
     "timeout": "1m"
   },
   "dependencies": {
-    "eslint": "^5.15.1",
-    "espree": "^5.0.1"
+    "acorn": "^6.1.1",
+    "acorn-walk": "^6.1.1"
   },
   "description": "webhint parser needed to analyze JavaScript files",
   "devDependencies": {
-    "@types/eslint": "^4.16.6",
+    "@types/acorn": "^4.0.5",
+    "@types/estree": "^0.0.39",
```

Comment:
I think you are missing `estree` dependency. You have the types and you are using it later in the code but I don't see it in the `package.json`. We are probably loading the `eslint` packed one.

Reply:
Nope, just need the types as `acorn` emits an `estree` compatible AST.

Reply:
That's going to be a problem once I start working with the package resolver, need to think how to solve it.

Reply:
Also worth noting that there isn't an `estree` package as far as I'm aware - just `@types/estree`.